util/src/main/java/org/datacommons/util/McfParser.java

```diff
@@ -504,15 +528,20 @@ public static SchemaTerm parseSchemaTerm(String value, BiConsumer<String, String
       List<CSVRecord> records = parser.getRecords();
       if (records.isEmpty()) {
         if (errCb != null) {
-          errCb.accept("StrSplit_EmptyToken", "Empty value found (" + arg.context + ")");
+          String prop_suffix = "";
+          if (argContext != null) {
+            prop_suffix = "_" + argContext.prop;
+          }
+          errCb.accept(
+              "StrSplit_EmptyToken" + prop_suffix, "Empty value found (" + argContextString + ")");
```

Comment:
In an existing test, maybe you could add a check like https://github.com/datacommonsorg/import/blob/master/util/src/test/java/org/datacommons/util/TmcfCsvParserTest.java#L46-L47 ?

Reply:
done